manifest.yml

```
@@ -0,0 +1,5 @@
+name : "coreMQTT"
+version: "v1.0.0"
+description: |
+  "Client implementation of the MQTT 3.1.1 specification for embedded devices.\n"
```

The [coreJSON repo](https://github.com/FreeRTOS/coreJSON) uses the "suitable for microcontrollers" phrase. Should the "About" text and this manifest description of this repository be also consistent with it?
```suggestion
  "Client implementation of the MQTT 3.1.1 specification for embedded devices.\n"
```